src/sentry/utils/redis.py

```diff
@@ -266,6 +266,41 @@ def call_script(client, keys, args):
         """.format(
             path
         )
-        return script(keys, args, client)
+        if not script:
+            script[0] = Script(client, resource_string("sentry", posixpath.join("scripts", path)))
```

Comment:
sorry

Reply:
Could potentially set the client back to None here to replicate old behaviour. Not sure it matters much